### ISSUE TITLE
Fix DAP issues for Select

### ISSUE THREAD

**Issue by youda97** [IBM/carbon-components-angular#116]
Level | Description
-- | --
V | All content must reside within a WAI-ARIA landmark or labelled region role.

**Comment by cal-smith** [comment]
1. Likely a label/input association issue

**Comment by cal-smith** [comment]
false positive, since the preview page doesn't have a `main` tag. Unfortunately there's no reasonable way for us to add one at the moment. Apps should of course render into a `main` or other valid region.
